src/Senders/Jaeger.Senders.Grpc/README.md

```diff
@@ -0,0 +1,39 @@
+# gRPC sender for Jaeger (Jaeger.Senders.GRPC)
+
+## Usage
+This package contains the `GrpcSenderFactory` for use with `SenderResolver`.
+
+To use this, include the following line before doing any other calls to the `Configuration` API.
+
+```C#
+Configuration.SenderConfiguration.DefaultSenderResolver = new SenderResolver(loggerFactory)
+	// You can add other ISenderFactory instances too
+	.RegisterSenderFactory<ThriftSenderFactory>();
+```
+
+If you need to set `JAEGER_SENDER_FACTORY` (when multiple factories are registered), `GrpcSenderFactory.Name` corresponds to `"grpc"`.
+
+## Configuration
+Independent on what variables are set in `Configuration.SenderConfiguration`, a `GrpcSender` is created.
+
+```C#
+new SenderConfiguration(loggerFactory)
+	// Used with UdpSender:
+	.WithGrpcTarget(grpcTarget)                    // Host and port or any other valid gRPC target
+	.WithGrpcRootCertificate(grpcRootCertificate); // Path to the root certificate
+```
+
+## Contained Senders
+This package contains one sender. `GrpcSender` for communicating with the `Jaeger Collector`.
+
+In containerized environments, you usually want to talk to the `Jaeger Agent` via UDP for fast communication with little overhead. This package does not allow communication with `Jaeger Agent`. So if you require that, use the `ThriftSenderFactory` instead.
+
+### GrpcSender
+Communicates with the `Jaeger Collector`. This sender offers authentication through client certificate using `GrpcRootCertificate`.
```

Comment:
I don't think it's accurate to refer to this as "client certificate" - root/CA cert only authenticates the signatory of the server cert. To support client cert, another pair must be provided in the config, e.g. https://grpc.github.io/grpc/csharp/api/Grpc.Core.SslCredentials.html#Grpc_Core_SslCredentials__ctor_System_String_Grpc_Core_KeyCertificatePair_

Reply:
You are right, it’s only the PEM content of the root certificate used to authenticate the server certificate. Is Jaeger collector allowing client certificate usage / allows for setting a server certificate? If yes, we might want to add also a way to really set the client certificate.

Reply:
collector only has this for clients:
```
--collector.grpc.tls.client-ca string             Path to a TLS CA (Certification Authority) file used to verify certificates presented by clients
```

Reply:
Fixed. Thanks :)